.env.example

```diff
@@ -1,2 +1,12 @@
 LOG_LEVEL="debug"
 DEV_MODE="true"
+CF_PRICE_CLASS="PRICE_CLASS_ALL"
+CF_AWS_WAF="arn:aws:wafv2:us-east-1:123456789012:global/webacl/ExampleWebACL/473e64fd-f30b-4765-81a0-62ad96dd167a"
+CF_CUSTOM_SSL_CERT="arn:aws:acm:us-east-1:450112707314:certificate/589a0d1e-70bd-4244-bc34-5eba5ad13fc7"
+CF_SECURITY_POLICY="TLSv1.2_2021"
+CF_ENABLE_LOGGING="true"
+CF_S3_BUCKET_LOG="gympass-staging-cloudfront-access-log.s3.amazonaws.com"
```

Comment:
Are these actual valid ARNs and bucket address from our accounts? We probably shouldn't keep these in version control

Reply:
It makes sense. I'm going to change it, tkss